packages/camera/camera_android_camerax/lib/src/android_camera_camerax.dart

```diff
@@ -122,7 +128,7 @@ class AndroidCameraCameraX extends CameraPlatform {
 
   /// The prefix used to create the filename for video recording files.
   @visibleForTesting
-  final String videoPrefix = 'MOV';
+  final String videoPrefix = 'REC';
```

Comment:
Changed this + the .temp --> .mp4 extension change to be consistent with camera_android, see [source](https://github.com/flutter/packages/blob/d670b2c38c8db0a773aa703e7d3f15682e05ad7f/packages/camera/camera_android/android/src/main/java/io/flutter/plugins/camera/Camera.java#L1250).

Reply:
Are we guaranteed that the file is an mp4? If not then maybe we should leave it .temp because having the wrong file extension is worse than having an unknown file extension.